gwpy/cli/cliproduct.py

```diff
@@ -695,9 +697,14 @@ def run(self):
         self.log(3, ('Verbosity level: %d' % self.verbose))
 
         self.log(3, 'Arguments:')
+        hastex = False
         argsd = vars(self.args)
         for key in sorted(argsd):
             self.log(3, '{0:>15s} = {1}'.format(key, argsd[key]))
+            if isinstance(argsd[key], str):
+                hastex |= '$' in argsd[key]
+        hastex |= self.args.mode == 'spectrum'
+        rc('text', usetex=hastex)
```

Comment:
@areeda, I'm not sure this is the right way to go, given that matplotlib has (some) support for mathtext without using latex. Can you experiment with always using functions like `label_to_latex` but _not_ using `text.usetex`?